index.js

```diff
@@ -193,3 +199,35 @@ function validateEmail(event) {
 }
 
 form.addEventListener('submit', validateEmail);
+
+// Local storage
+const username = document.getElementById('username');
+const mail = document.getElementById('mail');
+const message = document.getElementById('message');
+const formButton = document.getElementById('formButton');
+
+formButton.addEventListener('click', () => {
+  const nameValue = username.value;
+  const mailValue = mail.value;
+  const messageValue = message.value;
+
+  const user = {
+    nameValue,
+    mailValue,
+    messageValue,
+  };
+
+  if (nameValue && mailValue && messageValue) {
+    const stringedUser = JSON.stringify(user);
+    localStorage.setItem('user', stringedUser);
+  }
+});
```

Comment:
- [ ] Kindly be noticed that after filling the form, and not submitting it, the entered values should be saved in the browser.
- [ ] So kindly add an event when the input and textarea change the localstorage also change.